pennylane_lightning/_serialize.py

```diff
@@ -27,6 +27,7 @@
 from pennylane.grouping import is_pauli_word
 from pennylane.operation import Observable, Tensor
 from pennylane.tape import QuantumTape
+import pennylane as qml
```

Comment:
Should we use the same structure of 
```
from pennylane import matrix
```
or something like
```
from pennylane import matrix as qml_mat
```

to avoid pulling in the full library?

Reply:
Though it is probably fine.

Reply:
Ah! I fix it now.

Reply:
Hi @mlxd, would you check if this is completely fixed in both `_serialized` and `lightning_qubit`?

Reply:
Ok. Tests are failed! Because the `qml.matrix` is new and not supported with current versions of PL. So, we should wait till the next release of PL.